src/sdb.h

```diff
@@ -32,8 +32,10 @@ extern "C" {
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <io.h>
-extern void *__cdecl _aligned_malloc(size_t, size_t);
+extern __attribute__((dllimport)) void *__cdecl _aligned_malloc(size_t, size_t);
 extern char *strdup (const char *);
+#undef r_offsetof
+#define r_offsetof(type, member) ((unsigned long) (ut64)&((type*)0)->member)
```

Comment:
No. that definition is at line 37, the other one is at line 20.
i've just changed it due the cast warning (from 8 bytes to 4 bytes)